main/lsp/requests/rename.cc

```diff
@@ -38,6 +38,43 @@ bool isValidRenameLocation(const core::SymbolRef &symbol, const core::GlobalStat
     return true;
 }
 
+class LocalRenamer : public AbstractRenamer {
+public:
+    LocalRenamer(const core::GlobalState &gs, const LSPConfiguration &config, const string oldName,
+                 const string newName, std::vector<core::Loc> localUsages)
+        : AbstractRenamer(gs, config, oldName, newName), localUsages(localUsages) {
+        // If the name is the same as before or empty, return an error. VS Code already prevents this on its own, but
+        // other IDEs might not
+        if (newName.empty()) {
+            invalid = true;
+            error = "The new name must not be empty.";
```

Comment:
ooc, do we have a test for this?

Reply:
I can't seem to get a test working with an empty `newName` attribute. Is there some trick? Alternatively, we can also just remove this since VS Code won't allow empty names.

Reply:
In that case I’d rather just leave this behavior in and not test it.